src/components/sidebar/channel-members/ChannelMembers.js

```diff
@@ -7,27 +7,72 @@ import User from './User'
 export default class ChannelMembers extends PureComponent {
   static propTypes = {
     users: PropTypes.array.isRequired,
+    totalMembers: PropTypes.number,
     channel: PropTypes.object.isRequired,
     currUser: PropTypes.object.isRequired,
+    sidebarRef: PropTypes.oneOfType([PropTypes.func, PropTypes.object]),
     onLoad: PropTypes.func,
     onOpen: PropTypes.func,
     onKick: PropTypes.func,
   }
 
   static defaultProps = {
+    totalMembers: 0,
+    sidebarRef: undefined,
     onLoad: noop,
     onOpen: noop,
     onKick: noop,
-    users: [],
+  }
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      limit: 50,
```

Comment:
Let's move this out of state since it's never changing.

Reply:
Done.